recipes/lzma_sdk/9.20/conanfile.py

```diff
@@ -1,7 +1,9 @@
-from conans import ConanFile, tools, AutoToolsBuildEnvironment, VisualStudioBuildEnvironment
+from conan import ConanFile
+from conan.tools.files import get, chdir, replace_in_file, rm
+from conans import tools, AutoToolsBuildEnvironment, VisualStudioBuildEnvironment
 import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.52.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
tools.files.rm requires 1.50.0 only. 1.52.0 introduced [export_conandata_patches](https://docs.conan.io/en/latest/reference/conanfile/tools/files/patches.html#conan-tools-files-export-conandata-patches)